doc/rst/technotes/fortranInterop.rst

```diff
@@ -0,0 +1,36 @@
+.. _readme-fortranInterop:
+
+.. default-domain:: chpl
+
+========================
+Fortran Interoperability
+========================
+
+This README describes support for calling Chapel functions from Fortran code
+and vice-versa. The features described here are still a work in progress.
```

Comment:
I would split the last sentence into a `.. note::` or `.. warning::`, so that it pops more to the user